CONTRIBUTING.md

```diff
@@ -336,19 +336,14 @@ license](https://swift.org/LICENSE.txt).
 
 SwiftPM uses [swift-ci](https://ci.swift.org) infrastructure for its continuous integration testing. The bots can be triggered on pull-requests if you have commit access. Otherwise, ask one of the code owners to trigger them for you.
 
-To run smoke test suite with the trunk compiler and other projects use:
-
-```
-@swift-ci please smoke test
-```
 
 This is **required** before a pull-request can be merged.
 
 
 To run just the self-hosted test suite (faster turnaround times so it can be used to get quick feedback) use:
 
 ```
-@swift-ci please smoke test self hosted
+@swift-ci please test self hosted
```

Comment:
chore (blocking): The directives also appear to support `@swift-ci please test linux`, `@swift-ci please test windows`  and `@swift-ci please test macos`.  Can we include these directives in the documentation?